packages/google_maps_flutter/google_maps_flutter/example/ios/RunnerTests/FlutterMock.h

```diff
@@ -0,0 +1,19 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+@import Foundation;
+@import Flutter;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface MockRegistrar : NSObject <FlutterPluginRegistrar>
```

Comment:
Why are you mocking these manually instead of with OCMock?

Reply:
I've never used OCMock. Thanks for the tip!